components/user/EntryTypeGraph.jsx

```diff
@@ -121,6 +149,43 @@ const EntryTypeGraph = ({ data }) => {
           </ResponsiveContainer>
         </Col>
       </Row>
+      <Row style={{ width: "100%" }}>
+        <Col style={{ height: "300px" }} className="d-flex flex-column col-md-6 col-sm-12">
+          <h1>Playtime Breakdown</h1>
+          <ResponsiveContainer>
+            <PieChart width={300} height={250} className="mx-auto">
+              <Pie
+                dataKey="percOfTotal"
+                isAnimationActive={true}
+                data={playtimePieChartData}
+                cx="50%"
+                cy="50%"
+                outerRadius={90}
+                fill="#00d0ff"
+              >
+                {biomePieChartData.map((_, idx) => (
+                  <Cell key={`cell-${idx}`} fill={colourList[(idx + 2) % 4]} />
+                ))}
+              </Pie>
+              <Legend layout="horizontal" verticalAlign="bottom" align="right" />
+              <Tooltip />
```

Comment:
Tooltip needs to be formatted, see how its done in the bar graphs above 

![image](https://github.com/Specnr/ResetAnalytics/assets/74469583/3316c8f6-9dbf-40bc-b2f6-9ee03090c194)

Reply:
tooltip is now formatted, the first graph is cummulative